Jantimizer/Histograms/HistogramEquiDepth.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Histograms
+{
+    /// <summary>
+    /// Contains n buckets, which each represent the same m number of elements.

```

Comment:
As in each bucket contains the same elements, or that each bucket contains m number of elements?